virttest/utils_conn.py

```diff
@@ -2335,3 +2335,17 @@ def conn_setup(self):
         self.connect_to_unix_socket(tcp2unix_session, dest_pmsocat_path)
         self.add_firewall_ports(tcp2unix_session)
         LOG.debug("TCP2UNIX setup successfully.")
+
+
+def update_crypto_policy(val="DEFAULT", verbose=True, ignore_status=True, shell=True):
```

Comment:
Do you need return one boolean value indicating whether update succeed or not?  since the method name is update_crypto_policy, it is very natural that people think we need get one final result that whether the update is successful or not.

Reply:
Thanks, The result is returned now.